### ISSUE TITLE
Unable to resolve module `./src/App`: could not resolve as a file nor as a folder

### ISSUE THREAD

**Issue by brettdh** [goinverse-oss/app#3]
When running `npm start` and then `react-native run-ios`:
```
Bundling `index.js`  [development, non-minified]  0.0% (0/96), failed.
error: bundling failed: UnableToResolveError: Unable to resolve module `./src/App` from `/Users/brhiggins/src/theliturgists/app/index.js`: could not resolve `/Users/brhiggins/src/theliturgists/app/src/App' as a file nor as a folder
```
`src/App.jsx` definitely exists, and it's not a filename extension problem; I changed the import to add `.jsx` but it still failed. 🤔

**Comment by wittjosiah** [comment]
@brettdh Seems like we've stumbled upon a fairly widespread issue... https://github.com/facebook/react-native/issues/14209 None of the solutions have worked for me thus far, but I will keep trying to resolve it.

If anyone else has any success let me know.

**Comment by brettdh** [comment]
Wow, there is a lot of voodoo being traded in that conversation.

I’ll also try some things tomorrow.

**Comment by brettdh** [comment]
Fixed by #4.
